### PR TITLE
apache-fop: update to 2.1

### PR DESCRIPTION
**Pull request by rkallos**

### CODE REVIEW

File: srcpkgs/apache-fop/template [hunk 1]
```diff
@@ -10,8 +10,8 @@ distfiles="
  http://mirror.dkd.de/apache/xmlgraphics/fop/source/fop-${version}-src.tar.gz
```
Comment (TheDoctorsLife) [inline srcpkgs/apache-fop/template]: Remove the whitespace at the end of the url (its right after tar.gz).

Comment (rkallos) [inline srcpkgs/apache-fop/template]: Good catch! Sorry about that. Rebased.

File: srcpkgs/apache-fop/template [hunk 2]
```diff
@@ -46,7 +46,7 @@ do_install() {
 	vcopy LICENSE ${_docs}
 	vcopy NOTICE ${_docs}
 	vcopy README ${_docs}
-	vcopy status.xml ${_prefix}
+	vcopy build.xml ${_prefix}
```
Comment (Vaelatern) [inline srcpkgs/apache-fop/template]: What is the source of this rename?

Comment (rkallos) [inline srcpkgs/apache-fop/template]: Thanks for bringing this up. I forgot to mention, but it appears as though status.xml does not exist after building anymore. That said, build.xml was there. I suspect it was renamed.

```
$ ./xbps-src install apache-fop
---snip---
=> apache-fop-2.1_1: running do_install ...
cp: cannot stat 'status.xml': No such file or directory
=> ERROR: apache-fop-2.1_1: failed to run do_install() at line 174.
```

Comment (Gottox) [inline srcpkgs/apache-fop/template]: build.xml is used by ant, I don't think that status.xml is the replacement.

Comment (Gottox) [inline srcpkgs/apache-fop/template]: please check what happened to status.xml :)

**Comment by Gottox** [comment]
No progress for 8 month, closing

**Pull request closed without merging**